examples/oft-solana/README.md

```diff
@@ -283,6 +283,36 @@ npx hardhat lz:oapp:wire --oapp-config layerzero.config.ts
 
 With a squads multisig, you can simply append the `--multisig-key` flag to the end of the above command.
 
+### Wiring Solana to Aptos
```

Comment:
Up until this point I've worked to make the oft-solana example to have a streamlined experience. Moving to simple config was a devex upgrade. This PR is a great step towards having a unified OFT example, but given the lack of support for simple config, the Aptos wiring needs to be an aside for now. Best way I think is to put it into a new docs/Aptos.md (just for the README/docs)

The default instructions for partners that just want EVM + Solana only should not be changed until we can get Aptos supported by Simple Config.